<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to MARTA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtResources.qrc">
    <normaloff>:/newPrefix/ui/marta-logo.png</normaloff>:/newPrefix/ui/marta-logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {background-color: #ffffff} 
QLabel {color: #009fd7; font-weight: bold;}
#frameLabelTop {background-color: transparent; color: #ee9306; font-size: 30px;}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="manageCardsLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#manageCardsLabel {font: normal; text-decoration: underline; color: blue;}</string>
   </property>
   <property name="text">
    <string>Manage Cards</string>
   </property>
  </widget>
  <widget class="QLabel" name="endTripLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#endTripLabel {font: normal; color: blue; text-decoration: underline}</string>
   </property>
   <property name="text">
    <string>End Trip</string>
   </property>
  </widget>
  <widget class="QLabel" name="tripLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#tripLabel {font-weight: bold; color:rgb(57, 140, 78);}</string>
   </property>
   <property name="text">
    <string>No Active Trip</string>
   </property>
  </widget>
  <widget class="QComboBox" name="breezeCardBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="startAtLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Start at</string>
   </property>
  </widget>
  <widget class="QComboBox" name="startAtBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="viewTripHistoryButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View Trip History</string>
   </property>
  </widget>
  <widget class="QLabel" name="endingAtLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ending at</string>
   </property>
  </widget>
  <widget class="QComboBox" name="endingAtBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="breezeCardLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Breeze Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceAmount">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {font: normal; color: black;}</string>
   </property>
   <property name="text">
    <string>$ MONEY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="windowModality">
    <enum>Qt::ApplicationModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="frameLabelTop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {background-color: transparent}</string>
   </property>
   <property name="text">
    <string>Welcome to MARTA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qtResources.qrc"/>
 </resources>
 <connections/>
</ui>
